.github/workflows/CI.yml

```diff
@@ -17,23 +17,22 @@ concurrency:
   group: "${{ github.workflow }}-${{ github.ref }}"
   cancel-in-progress: true
 
+defaults:
+  run:
+    shell: bash -leo pipefail {0}    
+
 jobs:
   test:
     name: ${{ matrix.os }}, py-${{ matrix.python-version }}, OpenMM-${{ matrix.openmm }}, pymbar-${{ matrix.pymbar-version }}
     runs-on: ${{ matrix.os }}
     strategy:
       fail-fast: false
       matrix:
-        python-version: ["3.9", "3.10"]
-        openmm: ["7.7", "8.0"]
-        os: [macOS-latest, ubuntu-latest]
+        python-version: ["3.10", "3.11", "3.12"]
+        openmm: ["8.0", "8.1.1", "8.1.2"]
```

Comment:
Nonblocking comment. It's a great idea that we are testing with different OpenMM versions. I think for the longer term we should make sure we understand what kind of versioning system (semantic or not, etc.) is OpenMM using, such that we don't have to be testing with every minor OpenMM release out there. I hope this makes sense.

Reply:
Yes, I think post this merge we can probably drop 8.0, and make a decision if we want to support n, n-1 openmm releases or something else.

Reply:
Created an issue for this in https://github.com/choderalab/openmmtools/issues/747